api/templates/main.svg

```diff
@@ -26,9 +26,9 @@
     <!-- title -->
     <g transform="translate(10, {{ height - title_height - 50 }})">
         <text fill="{{ title_color }}" font-family="Roboto, Segoe UI, Ubuntu, Arial, sans-serif"
-            font-weight="600" font-size="15px">
+            font-weight="600" font-size="15px" direction="{{ 'rtl' if rtl_title else 'ltr' }}">
             {% for line in title_lines %}
-            <tspan x="0" dy="{{ title_line_height }}px">{{ line }}</tspan>
+            <tspan x="{{ width - 12 if rtl_title else 10 }}" dy="{{ title_line_height }}px">{{ line }}</tspan>
```

Comment:
It seems like the text doesn't line up with the stats, this should definitely (at least for ltr) be 0 to keep existing functionality the same.

The rtl offset *could* be changed here, but I think it may make more sense to change the 10 in the transform on line 27 instead since that is where the current offset is defined.

Let me know what you think.

Thanks!

![](https://ytcards-git-fork-vyvy-vi-feat-support-rtl-denvercoder1.vercel.app/?id=dQw4w9WgXcQ&title=%D7%9C%D7%95%D7%A8%D7%9D%20%D7%90%D7%99%D7%A4%D7%A1%D7%95%D7%9D%20%D7%93%D7%95%D7%9C%D7%95%D7%A8%20%D7%A1%D7%99%D7%98%20%D7%90%D7%9E%D7%98,%20%D7%A7%D7%95%D7%A0%D7%A1%D7%A7%D7%98%D7%95%D7%A8%D7%A8%20%D7%90%D7%93%D7%99%D7%A4%D7%99%D7%A1%D7%99%D7%A0%D7%92%20%D7%90%D7%9C%D7%99%D7%AA&lang=he&timestamp=1693000000&max_title_lines=2&width=250&border_radius=5)

![](https://ytcards-git-fork-vyvy-vi-feat-support-rtl-denvercoder1.vercel.app/?id=J7Fm7MdZn_E&title=Hosting+a+Python+Discord+Bot+for+Free+with+Fly.io&lang=en&timestamp=1661733947&background_color=%230d1117&title_color=%23ffffff&stats_color=%23dedede&width=250&border_radius=5&duration=403&max_title_lines=1)